src/types/EmbeddedPaymentElement.tsx

```diff
@@ -0,0 +1,427 @@
+  primaryButtonLabel?: string;
+  /** Optional configuration to display a custom message when a saved payment method is removed. iOS only. */
+  removeSavedPaymentMethodMessage?: string;
+  /** The list of preferred networks that should be used to process payments made with a co-branded card.
+   * This value will only be used if your user hasn't selected a network themselves. */
+  preferredNetworks?: Array<CardBrand>;
+  /** By default, EmbeddedPaymentElement will use a dynamic ordering that optimizes payment method display for the customer.
+   *  You can override the default order in which payment methods are displayed in EmbeddedPaymentElement with a list of payment method types.
+   *  See https://stripe.com/docs/api/payment_methods/object#payment_method_object-type for the list of valid types.  You may also pass external payment methods.
+   *  - Example: ["card", "external_paypal", "klarna"]
+   *  - Note: If you omit payment methods from this list, they’ll be automatically ordered by Stripe after the ones you provide. Invalid payment methods are ignored.
+   */
+  paymentMethodOrder?: Array<String>;
+  /** This is an experimental feature that may be removed at any time.
+   *  Defaults to true. If true, the customer can delete all saved payment methods.
+   *  If false, the customer can't delete if they only have one saved payment method remaining.
+   */
+  allowsRemovalOfLastSavedPaymentMethod?: boolean;
+  /**
+   * By default, EmbeddedPaymentElement will accept all supported cards by Stripe.
+   * You can specify card brands EmbeddedPaymentElement should block or allow payment for by providing an array of those card brands.
+   * Note: This is only a client-side solution.
+   * Note: Card brand filtering is not currently supported in Link.
+   */
+  cardBrandAcceptance?: PaymentSheetTypes.CardBrandAcceptance;
+  /** (Private Preview) This parameter is expected to be removed once we GA this feature
+   * When using customerSessions, allow users to update their saved cards
+   */
+  updatePaymentMethodEnabled?: boolean;
+  /** The view can display payment methods like “Card” that, when tapped, open a sheet where customers enter their payment method details.
+   * The sheet has a button at the bottom. `formSheetAction` controls the action the button performs.
+   */
+  formSheetAction?: EmbeddedFormSheetAction;
+  /** Controls whether the view displays mandate text at the bottom for payment methods that require it. If set to `false`, your integration must display `PaymentOptionDisplayData.mandateText` to the customer near your “Buy” button to comply with regulations.
+   * Note: This doesn't affect mandates displayed in the form sheet.
+   */
+  embeddedViewDisplaysMandateText?: boolean;
+  /**
+   * Determines the behavior when a row  is selected. Defaults to `.default`.
+   */
+  rowSelectionBehavior?: RowSelectionBehavior;
```

Comment:
As mentioned above, I don't think this will work on Android, and will need to be passed into init. Probably worth not doing it as a first implementation.